practice-two/src/index.html

```diff
@@ -48,7 +48,7 @@
     </div>
 
     <!-- Modal Add notes -->
-    <div class="modal" id="addnotesmodal" tabindex="-1" role="dialog" aria-labelledby="addnotesmodalTitle"
+    <form class="modal" id="addnotesmodal" tabindex="-1" role="dialog" aria-labelledby="addnotesmodalTitle"
```

Comment:
Should separate html structures and styles between forms and modals

Reply:
Yesss Sir!!!